content/200-concepts/100-components/08-prisma-engines/index.mdx

```diff
@@ -136,11 +137,11 @@ Format Binary        : prisma-fmt d6ff7119649922b84e413b3b69660e2f49e2ddf3 (at c
 
 </TabbedContent>
 
-### Hosting binaries
+### Hosting engines
 
-The [`PRISMA_BINARIES_MIRROR`](../../../reference/api-reference/environment-variables-reference#prisma_binaries_mirror) <span class="api"></span> environment variable allows you to host a binary via a private server, AWS bucket or other cloud storage.
-This can be useful if you have a custom OS which requires you to use your own custom built binaries.
+The [`PRISMA_ENGINES_MIRROR`](../../../reference/api-reference/environment-variables-reference#prisma_engines_mirror) <span class="api"></span> environment variable allows you to host engines via a private server, AWS bucket or other cloud storage.
+This can be useful if you have a custom OS which requires you to use your own custom built engines.
```

Comment:
```suggestion
This can be useful if you have a custom OS that requires custom-built engines.
```

Reply:
Hihi, another `-` - let's slip this one in.